### ISSUE TITLE
Case where "// input var okay" ignored

### ISSUE THREAD

**Issue by paulschreiber** [WordPress-Coding-Standards/WordPress-Coding-Standards#542]
I'm seeing a false positive for SuperGlobalInputUsage.AccessDetected / a case where `// input var okay` is ignored:

```
----------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
----------------------------------------------------------------------
 25 | WARNING | Detected access of super global var $_GET, probably need
    |         | manual inspection.
    |         | (WordPress.VIP.SuperGlobalInputUsage.AccessDetected)
----------------------------------------------------------------------
```

Fails
```
		<?php
		if ( isset( $_GET['updated'] ) ) { // input var okay ?>
```

Passes
```
		<?php
		if ( isset( $_GET['updated'] ) ) { // input var okay
		?>
```

**Comment by jrfnl** [comment]
@paulschreiber Could you please test with the change I made in #619 ? Thanks.
